src/main/resources/languages/languages_de_DE.properties

```diff
@@ -4,6 +4,9 @@ startLogin=ANMELDUNG \u00d6FFNEN
 username=Benutzername
 password=Passwort
 repeatPassword=Passwort wiederholen
+changePassword=Passwort \u00e4ndern
+newPassword=Neues Passwort
```

Comment:
Please make sure all newly introduced identifier are as well inserted into to the src/main/resources/languages/languages.properties file.

Reply:
fixed